### PR TITLE
adding rixs recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: 
https://github.com/nsls-ii-forge/rixs-feedstock
https://github.com/nsls-ii-forge/sixtools-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

**Comment by kbeeperez** [comment]
@conda-forge/staged-recipes should be ready to merge now.

**Comment by kbeeperez** [comment]
Attn @mrakitin is there anything that needs changing?

File: recipes/rixs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - matplotlib
```
Comment (dopplershift) [inline recipes/rixs/meta.yaml:23]:
```suggestion
    - matplotlib-base
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
@mrakitin are you ok being listed as a maintainer?

**Review by dopplershift — APPROVED** [review]

File: recipes/rixs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{% set name = "rixs" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: '{{ name|lower }}'
```
Comment (mrakitin) [inline recipes/rixs/meta.yaml:5]:
```suggestion
  name: {{ name|lower }}
```

File: recipes/rixs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  version: '{{ version }}'
```
Comment (mrakitin) [inline recipes/rixs/meta.yaml:6]:
```suggestion
  version: {{ version }}
```

File: recipes/rixs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c563f92f34db46cd9d01ac1ba3e9c7b41cf206d42dc635bf031ad8ec1cbc0db
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (mrakitin) [inline recipes/rixs/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

**Review by mrakitin — APPROVED** [review]
Looks good to me, I added a couple of "cosmetic" suggestions.

@conda-forge/staged-recipes, I agree to maintain this package.

File: recipes/sixtools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 38950e920b9ae28979f1dee66756763714b601ed1553ec864a8278de6c295105
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```
Comment (mrakitin) [inline recipes/sixtools/meta.yaml:15]: Making the install comamnd consitent:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

**Comment by mrakitin** [comment]
As another recipe was added here, I agree to maintain both `sixtools` and `rixs`.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**